config/300-channel.yaml

```diff
@@ -44,6 +47,9 @@ spec:
     - name: Reason
       type: string
       JSONPath: ".status.conditions[?(@.type==\"Ready\")].reason"
+    - name: Hostname
+      type: string
+      JSONPath: .status.address.hostname
```

Comment:
I think we want to switch to the URI, rather than hostname soonish. Either change it now or add a comment here to change it later.

Reply:
should it be .status.address.url ?

Reply:
Changed it to .status.address.url, thanks guys!